infra/type-overrides.d.ts

```diff
@@ -11,6 +11,12 @@ interface IDBObjectStore {
   openKeyCursor(query?: IDBValidKey | IDBKeyRange | null, direction?: IDBCursorDirection): IDBRequest<IDBCursor | null>;
 }
 
+// Service Worker
+// ------------------------------------------------------------------------- //
+interface CacheQueryOptions {
```

Comment:
There's a [few more](https://developers.google.com/web/updates/2016/09/cache-query-options): `ignoreSearch`, `cacheName`, `ignoreMethod`, and `ignoreVary`.

I guess only `cacheName` is needed for this PR, but might as well add them all?

Reply:
The built-in TS types includes all those. The only one it was missing was `cacheName` for some reason (perhaps because it shares the same type between `caches.match()` and `cache.match()`?).